Makefile

```diff
@@ -40,7 +41,7 @@ else
 		DONT_BUILD_LZHAM ?= 1
 		DONT_BUILD_CSC ?= 1
 	else
-		LDFLAGS	+= -lrt -static
```

Comment:
This breaks Travis tests with ARM and PowerPC (https://travis-ci.org/inikep/lzbench/builds/229271314)